doc/Language/nativecall.pod6

```diff
@@ -653,9 +653,18 @@ say $esponja;
 
 NativeCall also supports native functions that take functions as arguments.  One
 example of this is using function pointers as callbacks in an event-driven
-system.  When binding these functions via NativeCall, one need only provide the
+system.  When binding these functions via NativeCall, one should need only provide the
 equivalent signature as L<a constraint on the code parameter|/type/Signature#Constraining_signatures_of_Callables>:
 
+    # See below--this syntax does not work on Rakudo currently
```

Comment:
I really see no point in including an example of something that does not work. We can mark an issue as "external" and wait for it to be solved instead.